notebooks/MAST/TESS/interm_gi_query/requirements.txt

```diff
@@ -0,0 +1,2 @@
+astropy>=3.0.4
+astroquery>=0.3.8
```

Comment:
Recommend upping the required astroquery to 0.3.9, it include many bugfixes to the mast module.

Reply:
Fixed.